content/api/transmissions.apib

```diff
@@ -3,1175 +3,798 @@ title: Transmissions
+                    {
+                        "address": {
+                            "email": "wilma@flintstone.com",
+                            "name": "Wilma Flintstone"
+                        },
+                        "substitution_data": {
+                            "customer_type": "Platinum",
+                            "year": "Freshman"
+                        }
+                    }
+                ],
+                "content": {
+                    "from": {
+                        "name": "Fred Flintstone",
+                        "email": "fred@flintstone.com"
+                    },
+                    "subject": "Big Christmas savings!",
+                    "reply_to": "Christmas Sales <sales@flintstone.com>",
+                    "headers": {
+                        "X-Customer-Campaign-ID": "christmas_campaign"
+                    },
+                    "html": "<p>Hi {{address.name}} \nSave big this Christmas in your area {{place}}! \nClick http://www.mysite.com and get huge discount\n</p><p>Hurry, this offer is only to {{user_type}}\n</p><p>{{sender}}</p>"
+                }
             }
-          ],
-          "content": {
-            "email_rfc822": "Content-Type: text\/plain\r\nTo: \"{{address.name}}\" <{{address.email}}>\r\n\r\n Hi {{first_name}} \nSave big this Christmas in your area {{place}}! \nClick http://www.mysite.com and get huge discount\n Hurry, this offer is only to {{customer_type}}\n {{sender}}\r\n"
-          }
-        }
 
 + Response 200 (application/json)
 
-  + Body
-
-            {
-              "results": {
+        {
+            "results": {
                 "total_rejected_recipients": 0,
-                "total_accepted_recipients": 2,
+                "total_accepted_recipients": 1,
                 "id": "11668787484950529"
-              }
             }
+        }
 
 + Response 400 (application/json)
 
         {
-          "errors" : [
-            {
-              "description" : "Unconfigured or unverified sending domain.",
-              "code" : "7001",
-              "message" : "Invalid domain"
-            }
-          ]
+            "errors": [
+                {
+                    "description": "Unconfigured or unverified sending domain.",
+                    "code": "7001",
+                    "message": "Invalid domain"
+                }
+            ]
         }
 
-+ Request Create Transmission Using CC Header (application/json)
 
-    + Headers
+### Send a Template [POST]
 
-            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+The following attributes should be set in the `content` object when using a template as the transmission's content:
 
-    + Body
++ Data Structure: Content Attributes
+    + template_id (string, required) - ID of the template to use.
+    + use_draft_template (boolean) - Whether to use the draft version. If this is set to `true` and no draft exists, the transmission will fail.
```

Comment:
"If this is set to `true` and no draft exists, the transmission will fail." but is that even possible

Reply:
I also thought this wasn't possible, but I checked with engineering. If you create a template with `published` as `true` no draft will exist.